tests/test_stdlib.py

```diff
@@ -495,9 +495,9 @@ def test_foreign_delegate(self, configure_for_pf, capsys):
 
         assert ("", "foo [in test_foreign_delegate]\n") == capsys.readouterr()
 
-    def test_clears_args(self, capsys, configure_for_pf):
+    def test_clears_args(self, configure_for_pf, capsys):
         """
-        We render our log records before sending it back to logging.  Therefore
+        We render our log records before sending it back to logging. Therefore
```

Comment:
I guess that double space was mistake. If that's not the case I'll remove the change as well.